src/manage-apps/fix-packages.md

```diff
@@ -0,0 +1,68 @@
+# Cleaning Up Packages with Terminal Commands
```

Comment:
I'm wondering if a title like `Package Manager Maintenance` could be simpler, potentially more positive, and short enough to have the same name in the sidebar and on the page itself.

Reply:
Changed to suggested title and changed section title in SUMMARY.md